docs/reprozip.rst

```diff
@@ -1,10 +1,10 @@
+The truth is computational reproducibility can be very painful to achieve for a number of reasons. Take the author-reviewer scenario of a scientific paper as an example. Authors must generate a guidebook that encapsulates all the inputs needed to correctly reproduce their experiments: the data, a complete specification of the experiment and its steps, and information about the originating computational environment (OS, hardware architecture, and library dependencies). Keeping track of this information manually is rarely feasible: it is both time-consuming and error-prone. First, computational environments are complex, consisting of many layers of hardware and software, and the configuration of the OS is often hidden. Second, tracking library dependencies is challenging, especially for large experiments. If authors did not plan for reproducibility since the beginning of the project, reproducibility is drastically hampered.
 
-For reviewers, even with a compendium in their hands, it may be hard to reproduce the results. There may be no instructions about how to execute the code and explore it further; the experiment may not run on his operating system; there may be missing libraries; library versions may be different; and several issues may arise while trying to install all the required dependencies, a problem colloquially known as `dependency hell <http://en.wikipedia.org/wiki/Dependency_hell>`__.
+For reviewers, even with a guidebook in their hands, it may be hard to reproduce the results. There may be no instructions about how to execute the code and explore it further; the experiment may not run on his operating system; there may be missing libraries; library versions may be different; and several issues may arise while trying to install all the required dependencies, a problem colloquially known as `dependency hell <http://en.wikipedia.org/wiki/Dependency_hell>`__.
 
 ReproZip helps alleviate these problems by allowing the user to easily capture all the necessary components in a single, distributable package. Also, the tool makes it easier to reproduce an experiment by providing different unpacking methods and interfaces that avoids the need to install all the required dependencies and that makes it possible to run the experiment under different inputs.
```

Comment:
I really meant compendium rather than guidebook -- I wanted to say that,
even with all the software packed in their hands, it is still hard to
reproduce (lack of instructions, missing libraries, OS, etc.).

On Wed, Sep 30, 2015 at 9:29 PM, Remi Rampin <notifications@github.com>
wrote:

> In docs/reprozip.rst
> <https://github.com/ViDA-NYU/reprozip/pull/150#discussion_r40872126>:
>
> >
> > -For reviewers, even with a compendium in their hands, it may be hard to reproduce the results. There may be no instructions about how to execute the code and explore it further; the experiment may not run on his operating system; there may be missing libraries; library versions may be different; and several issues may arise while trying to install all the required dependencies, a problem colloquially known as `dependency hell <http://en.wikipedia.org/wiki/Dependency_hell>`__.
> > +For reviewers, even with a guidebook in their hands, it may be hard to reproduce the results. There may be no instructions about how to execute the code and explore it further; the experiment may not run on his operating system; there may be missing libraries; library versions may be different; and several issues may arise while trying to install all the required dependencies, a problem colloquially known as `dependency hell <http://en.wikipedia.org/wiki/Dependency_hell>`__.
> >
> >  ReproZip helps alleviate these problems by allowing the user to easily capture all the necessary components in a single, distributable package. Also, the tool makes it easier to reproduce an experiment by providing different unpacking methods and interfaces that avoids the need to install all the required dependencies and that makes it possible to run the experiment under different inputs.
>
> I'm not sure this is what @fchirigati <https://github.com/fchirigati>
> meant by "compendium"
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ViDA-NYU/reprozip/pull/150/files#r40872126>.
>



-- 
Fernando Chirigati
Research Assistant and Doctoral Candidate at NYU
http://bigdata.poly.edu/~fchirigati/ <http://vgc.poly.edu/~fchirigati/>